samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
> The [ABNF TextMate grammar](https://github.com/sanssecours/ABNF.tmbundle/blob/master/Syntaxes/ABNF.tmLanguage) will also (correctly) mark parts of the original `tranquil.abnf` grammar as incorrect.

Judging by what the grammar marks as invalid, I'd say it's highlighting anything that isn't a whitelisted character (which excludes single-quotes).

The point of the samples is to give a realistic indication of what to expect. And, as you said, `.abnf` files are intended primarily for documentation-specific purposes, which means we can't (and shouldn't) expect every author to feel beholden by what the standard recommends.